Dockerfile

```diff
@@ -94,6 +94,9 @@ RUN touch ~/.profile && \
     >> ~/.profile
 
 EXPOSE 80/tcp
+EXPOSE 4369/tcp
```

Comment:
remember that there EXPOSE settings do not have effeect on Fly.io since braidnet is not runned as a container.

They can stay in case we want to run this repo as a container, but I would add a comment to underline this aspect.